content/php/concepts/arrays/terms/array-push/array-push.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'array_push()'
+Description: 'Inserts one or more element values to the end of an array, and return the new number of element values in the array.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_push()`** method inserts one or more element values to the end of an array, and return the new number of element values in the array.
+
+## Syntax
+
+```pseudo
+array_push($array, $value)
+```
+
+The `array_push()` function has one required parameter and some optional parameters:
+
+- `$array`: Specifies the input `array`.
+- `$value`: Specifies the element value to add. This function can add more than one element value at a time that can be seperated by commas.
+
+The `array_push()` function returns an `array` containing all element values from the given `$array` with new element values that has been inserted to the end of the `$array`.
+
+## Example
+
+The following example uses the `array_push()` function to add `"Chervolet"`, and `"Toyota"` element values to the end of the indexed array `$car_brands`:
+
+```php
+<?php
+$car_brands = array("Mercedes","Ford");
+array_push($car_brands,"Chervolet","Toyota");
+print_r($car_brands);
+?>
+```
+
+The example will result in the following output:
+
+```shell
+Array (
+[0] => Mercedes
+[1] => Ford
+[2] => Chervolet
+[3] => Toyota
+)
+```
+
+## Codebyte Example
```

Comment:
The codebyte is the same as the example, but for the values used. It should be a little different. Maybe you can do something like using `array_push()` a couple of times and printing the return value each time.